db/fixtures/miq_user_roles.yml

```diff
@@ -278,6 +278,7 @@
   - vm_infra_explorer
   - vm_html5_console
   - vm_vmrc_console
+  - vm_management_console
   - vm_native_console
```

Comment:
do we still want the `vm_native_console` entries?

ugh. this probably requires a migration and everything.

Reply:
Yes because vm_native_console is still a valid feature (ovirt/rhv vm console), it was being overloaded by something else (web browser details page)